core/management/commands/update_gitlab_webhooks.py

```diff
@@ -20,7 +21,7 @@ def add_arguments(self, parser: CommandParser) -> None:
 
     def handle(self, *args, **options):
         repos = Repository.objects.filter(
-            Q(author__service="gitlab") & ~Q(hookid=None),
+            Q(author__service="gitlab") & ~Q(hookid=None) & Q(webhook_secret=None),
```

Comment:
Do we still need/want to rely on `hookid`? Are we deprecating that eventually?

Reply:
No, we'll continue using that field.  It stores the external ID of the webhook installed in the Git provider's repo.  We need that ID to make any updates to the webhook.